emcc.py

```diff
@@ -3031,9 +2995,6 @@ def consume_arg():
       options.valid_abspaths.append(newargs[i + 1])
       newargs[i] = ''
       newargs[i + 1] = ''
-    elif newargs[i] == '--separate-asm':
```

Comment:
Keep this around and issue an appropriate error?

Reply:
Ah, good point, this PR needs to do that to be NFC, as we errored before. Fixing...